tools/jslint.js

```diff
@@ -215,10 +215,10 @@ if (cluster.isMaster) {
     const elapsed = process.hrtime(startTime)[0];
     const mins = padString(Math.floor(elapsed / 60), 2, '0');
     const secs = padString(elapsed % 60, 2, '0');
-    const passed = padString(successes, 6, ' ');
-    const failed = padString(failures, 6, ' ');
+    const passed = padString(successes, 4, ' ');
+    const failed = padString(failures, 4, ' ');
```

Comment:
No, it's [created on the fly](https://github.com/nodejs/node/blob/1e4674ae0aec78f32c26a297ac11b45d46c24344/benchmark/module/module-loader.js#L18-L31) when the `module-loader` benchmark is ran.